packages/angular/src/generators/utils/angular-version-utils.ts

```diff
@@ -1,10 +1,12 @@
 import type { Tree } from '@nrwl/devkit';
```

Comment:
@Coly010 moved these helpers because they are specific to generators and to avoid conflict with similar helpers for executors.

Reply:
Makes sense!